### ISSUE TITLE
Exception during deserialization

### ISSUE THREAD

**Issue by d4v1s** [EricZimmerman/KapeFiles#320]
KAPE version 0.9.5.0

Google isn't being much help with the deserialization error so I'm hoping you can point me in the right direction.

This doesn't happen with the 0.9.0.1 version, which is the only other one I have.  I can run a sync and open the GUI with zero problems.

----------

When I try launching the GUI using 0.9.5.0, I get this error as a popup: "Exception during deserialization. Verify syntax and try again."

It throws this error on these files:

Targets\!BasicCollection.tkape
Targets\!SANS_Triage.tkape
Targets\Apps\CloudStorage.tkape
Targets\Apps\Exchange.tkape
Targets\Browsers\WebBrowsers.tkape
Targets\Misc\KapeTriage.tkape
Targets\Misc\MiniTimelineCollection.tkape
Targets\Misc\RemoteAdmin.tkape
Targets\Misc\VirtualDisks.tkape
Targets\Windows\CombinedLogs.tkape
Targets\Windows\EvidenceOfExecution.tkape
Targets\Windows\FileSystem.tkape
Targets\Windows\Recycle.tkape
Targets\Windows\RecycleBinContent.tkape
Targets\Windows\RecycleBinMetadata.tkape
Targets\Windows\RegistryHives.tkape
Targets\Windows\WindowsNotifcationsDB.tkape

----------

It also happens when running a sync:

C:\tools\dfir\kape-newest> .\kape.exe --sync --debug --trace
Enabled debug messages...
Enabled trace messages...
KAPE version 0.9.5.0 Author: Eric Zimmerman (kape@kroll.com)

Checking for updated targets and modules at https://github.com/EricZimmerman/KapeFiles...

Reading 'C:\tools\dfir\kape-newest\Targets\!BasicCollection.tkape'
There was an error checking for updates: (Line: 8, Col: 9, Idx: 158) - (Line: 8, Col: 9, Idx: 158): Exception during deserialization

Use --debug mode for more details

----------

Windows 10 
- version 1909 
- build 18363.1198 (the latest this laptop will support)

.NET version 4.8.03752
- release 528040
Windows 10 May 2019 Update and Windows 10 November 2019 Update: 528040

----------

**Comment by rathbuna** [comment]
Have you tried deleting the Targets and Modules folders and resyncing again? If not, do that and report back.

**Comment by d4v1s** [comment]
I tried a million things but I did not try that and it looks like that fixed it.  Thank you!

----------

.\kape.exe --sync
KAPE version 0.9.5.0 Author: Eric Zimmerman (kape@kroll.com)

Checking for updated targets and modules at https://github.com/EricZimmerman/KapeFiles...

Updates found!

New targets
!Disabled\AppData
!Disabled\DirectoryTraversalWildCardExample
!Disabled\LiveUserFiles
...
timelining\RegTime_Software_TLN
timelining\RegTime_System_TLN
timelining\RegTime_UsrClass_TLN

NOTE: Ensure you place the necessary executables in the Modules\bin directory!

----------

And gkape runs without a problem.

----------

Thank you again!
